src/screens/HomePage/HomePage.tsx

```diff
@@ -67,15 +67,15 @@ export default function HomePage() {
   const showRisingHospitalizations =
     location.hash === '#explore-hospitalizations';
   const risingHospitalizationStates = [
+    'DE',
+    'IL',
     'MD',
     'MI',
     'MN',
     'NJ',
-    'WV',
-    'CT',
-    'IL',
-    'MA',
-  ]; // Updated on 29 March 2021
+    'PA',
+    'PR',
+  ]; // Updated on 7 April 2021 -- this is an illustrative list but not exhaustive
```

Comment:
We could add several more states, but the graph just gets noisy, hurts performance (we're downloading the API endpoint for each state), and also some of the "low but rising" states just don't really pop when on a normalized graph with the other ones, so I don't think we should try to make this be an exhaustive list...

Reply:
Agreed that we can treat this as a representative, not exhaustive, list.